pre-docs/inventory_version2.md

```diff
@@ -577,7 +577,7 @@ targets:
         user: root
         password:
           _plugin: pkcs7
-          encrypted-value: |
+          encrypted_value: |
```

Comment:
There is another `encrypted-value` at https://github.com/puppetlabs/bolt/pull/1215/files#diff-b2ec0c199c46859d3a750491f05091c2R566